src/util/safelywritablefile.cpp

```diff
@@ -31,7 +39,9 @@ const int kWindowsSharingViolationSleepBeforeNextRetryMillis = 100;
 
 } // namespace
 
-SafelyWritableFile::SafelyWritableFile(QString origFileName, bool useTemporaryFile) {
+SafelyWritableFile::SafelyWritableFile(QString origFileName,
+        bool createTemporaryFileCopy,
+        bool useSuffixIfNotPrefix) {
```

Comment:
This sounds if the parameter is used constitutional. Is this the case? I do not understand the IfNotPrefix part.

Reply:
I wanted to make it descriptive, if it is true that uses suffix, if not uses prefix.

At first, I named it `useSuffix`, but when it is passed as false, it sounds like we are not going to use suffix, so that sounded like we are using nothing. 

I also thought, I can name it `useSuffix`, but if it is passed false in any other class, we could have a bool constant which says, `usePrefix`

Reply:
Now with the last changes naming is more logical IMHO